app/views/shared/_add_repos_list.html.slim

```diff
@@ -1,4 +1,4 @@
-- if repos.size == 0
+- if repos.blank? || repos.size == 0
```

Comment:
The `blank?` method will cover when an array is empty.

```
[].blank?
=> true
```

We could remove the `repos.size == 0 ` check.